common/app/services/OphanApi.scala

```diff
@@ -105,3 +105,6 @@ object OphanApi extends ExecutionContexts with Logging with implicits.WSRequests
   def getMostViewedVideos(hours: Int, count: Int): Future[JsValue] =
     getBody("video/mostviewed")(Map("hours" -> hours.toString, "count" -> count.toString))
 }
+
+object OphanApi extends OphanApi(WS.client) //Do not use. TODO: To delete once we find an elegant way to inject OphanApi into SurgingContentAgent
+
```

Comment:
As explain in the PR description, I have not found a nice way to deal with the `SugingContentAgent` dependency within `meta`. 😞  Keeping this object for now